### DISCUSSION TITLE
Cloudflare's Browser Check opening in a modal

### DISCUSSION THREAD

**Discussion by BerkanYildiz** [laravel/nova-issues#4221] [General]
- Laravel Version: 9
- Nova Version: 4.4.2
- PHP Version: 8.0.18
- Database Driver & Version: Eloquent / MariaDB (Latest)
- Operating System and Version: Debian (Latest)
- Browser type and version: Firefox (Latest)

### Description:
When I'm on my dashboard and go away for a few minutes or so and Cloudflare wants to check my browser again, if I click on any links, it will show as a modal.

### Detailed steps to reproduce the issue on a fresh Nova installation:
- Go to your main dashboard.
- On Cloudflare's website, Turn On Cloudflare to its Highest security settings so it checks your browser when visiting the website for the first time.
- On your dashboard, click on anything in the menu.

![09d-20-22-57-1065](https://user-images.githubusercontent.com/15068742/167473703-814b7fae-aed1-4809-803b-00b81153cc54.png)

https://user-images.githubusercontent.com/15068742/167476399-4f8a508f-c9ba-478c-ac1a-9d40a2c990c7.mp4

### Side notes:
Technically I'm guessing this is the intended behaviour ? Any internal links that don't match nova's routes open as a modal it looks like.

**Comment by crynobone** [comment]
I believe Cloudflare's Browser Check might tamper with Inertia response headers which cause it to think it's an external link. This should be reported to Inertia Repository: https://github.com/inertiajs/inertia/issues
